src/endless/Analytics.cpp

```diff
@@ -331,6 +332,7 @@ void Analytics::urlEncode(const CString &in, CString &out)
 
 void Analytics::prefixId(CString &id)
 {
-	id.Format(_T("v=1&tid=%s&cid=%s&an=Endless%%20Installer&av=%s&ul=%s&cd1=%s&"),
-		m_trackingId, m_clientId, _T(RELEASE_VER_STR), m_language, m_windowsVersion);
+	id.Format(_T("v=1&tid=%s&cid=%s&an=Endless%%20Installer&av=%s&ul=%s&cd1=%s&cd2=%s"),
+		m_trackingId, m_clientId, _T(RELEASE_VER_STR), m_language, m_windowsVersion,
+		CEndlessUsbToolDlg::IsCoding() ? "code" : "eos");
```

Comment:
These need to be wide strings, and you're missing the trailing `&`:

```
11:07:03 - Analytics req: v=1&tid=UA-61784217-3&cid=ffa20047-8a44-4010-820b-223726d5dd26&an=Endless%20Installer&av=3.0.11.1-23-g3f8ba4a&ul=en-US&cd1=Windows%2010%2064-bit%20(Build%2014393)&cd2=??t=screenview&cd=DualBootInstallPage&sc=start
11:07:03 - Analytics: response code 200
11:07:03 - Analytics: submitted an invalid hit: {
  "hitParsingResult": [ {
    "valid": false,
    "parserMessage": [ {
      "messageType": "ERROR",
      "description": "A value is required for parameter 'dp'. Please see http://goo.gl/a8d4RP#dp for details.",
      "messageCode": "VALUE_REQUIRED",
      "parameter": "dp"
    } ],
    "hit": "/debug/collect?v=1\u0026tid=UA-61784217-3\u0026cid=ffa20047-8a44-4010-820b-223726d5dd26\u0026an=Endless Installer\u0026av=3.0.11.1-23-g3f8ba4a\u0026ul=en-US\u0026cd1=Windows 10 64-bit (Build 14393)\u0026cd2=??t=screenview\u0026cd=DualBootInstallPage\u0026sc=start"
  } ],
  "parserMessage": [ {
    "messageType": "INFO",
    "description": "Found 1 hit in the request."
  } ]
}
```

Notice in the URL: `&cd2=??t=screenview&`